shell/common/common_spark.sh

```diff
@@ -193,13 +193,14 @@ get_spark_substitutions() {
     SPARK_MAJOR_VERSION "$SPARK_MAJOR_VERSION" \
     SPARK_MEMORY_OVERHEAD "$SPARK_MEMORY_OVERHEAD" \
     EXECUTOR_MEM "$EXECUTOR_MEM" \
-    EXPERIMENT_ID "$EXPERIMENT_ID"
+    EXPERIMENT_ID "$EXPERIMENT_ID,$1"
 }
 
 get_spark_conf_dir() {
   echo -e "$(get_local_bench_path)/spark_conf"
 }
 
+# $1: Benchmark name
```

Comment:
I'm not completely against these types of "documentation comments" but we can also make the code clearer by assigning to a local variable `display_name=$1`.